develop/tutorials/articles/02-tooling/06-liferay-sample-modules/00-intro.markdown

```diff
@@ -0,0 +1,75 @@
+# Liferay Sample Modules
+
+Liferay provides working examples of sample modules that target different
+integration points in @product@. These working examples can be copy/pasted into
+your own independent project so you can take advantage of various Liferay
+extension points. Most of the sample modules include a single class using
+Declarative Services. Therefore, each sample is developed to demonstrate a
+single extension point and includes its own build files. Liferay's sample
+modules can be found in the
+[liferay-blade-samples](https://github.com/liferay/liferay-blade-samples)
+repository on Github.
+
+If you'd like to browse the repo locally or copy sample projects into your own
+project, [fork](https://help.github.com/articles/fork-a-repo/) and
+[clone](https://help.github.com/articles/cloning-a-repository/) the
+`liferay-blade-samples` repository.
+
+At first glance, you'll notice that the repository is broken up into four
+primary folders:
+
+- `bndtools`
+- `gradle`
+- `liferay-gradle`
+- `maven`
+
+The provided sample modules are organized into several different build chains to
+cater to a variety of developers. Each folder offers a set of Liferay projects
+that can be bootstrapped to that specific development environment. For example,
+the sample project residing in the `gradle` folder can be bootstrapped to a
```

Comment:
We should somehow recommend the use of this plugin for gradle users.  but tell them that we make the other one available for the instance where they can't adopt the liferay-gradle plugin.  Because the liferay-gradle plugin takes over the standard gradle 'jar' plugin, there may be cases where it is incompatible with the user's existing gradle build configuration.